content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -249,7 +249,7 @@ networking:
   podSubnet: ""
   serviceSubnet: 10.96.0.0/12
 nodeRegistration:
-  criSocket: /var/run/dockershim.sock
+  criSocket: /var/run/containerd/containerd.sock
```

Comment:
I'd leave this as is as docker is still a default runtime.

Reply:
Sounds good, thanks!